### PR TITLE
feat: entity

### PR DESCRIPTION
**Pull request by menduz**
The objective of having this entity as a common schema is to 

1. prevent accidental modification
2. and help deleting the `dcl-catalyst-common` repository by reducing it's responsibilities

### CODE REVIEW

File: src/platform/entity.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+ * @public
+ */
+export type Entity = {
+  /** @deprecated ADR45 removed entity versions. */
+  version: string
+  id: IPFSv1 | IPFSv2
+  type: EntityType
+  pointers: string[]
+  timestamp: number
+  content: ContentMapping[]
```
Comment (pedrotambo) [inline src/platform/entity.ts:30]: is it safe to change this field from optional to mandatory?

Comment (agusaldasoro) [reply src/platform/entity.ts:30]: I'd rather leave it optional as it is now, and then we can enforce it in the future if needed

Comment (menduz) [reply src/platform/entity.ts:30]: I have two questions:
- in which cases this is optional?
- why do we have Deployment and Entity as separated things in the catalyst? they share the same fields but with different names

Comment (pedrotambo) [reply src/platform/entity.ts:30]: a) I dunno know. Just in the `catalyst-commons` is marked as optional. But if we think that it's always present (or should be), i'm in favor to mark it mandatory, and fix if something breaks. 
b) Which `Deployment` type do you refer, `DeploymentWithAuthChain`?

Comment (pedrotambo) [reply src/platform/entity.ts:30]: a) I dunno know. Just in the `catalyst-commons` is marked as optional. But if we think that it's always present (or should be), i'm in favor to mark it mandatory, and fix if something breaks. 
b) Which `Deployment` type do you refer, `DeploymentWithAuthChain`?

Comment (menduz) [reply src/platform/entity.ts:30]: Nope, just Deployment

Comment (pedrotambo) [reply src/platform/entity.ts:30]: This one https://github.com/decentraland/catalyst-commons/blob/master/src/types.ts#L88? 
Fortunately, it's only used in test as i see. We should delete it

**Pull request merged**